cherry_picker/cherry_picker.py

```diff
@@ -196,7 +196,7 @@ def sorted_branches(self):
     @property
     def username(self):
         cmd = ["git", "config", "--get", f"remote.{self.pr_remote}.url"]
-        result = self.run_cmd(cmd, required_real_result=True)
+        result = self.run_cmd(cmd, required_real_result=True).strip("\n")
```

Comment:
Strip any whitespace junk that might turn up? 
```suggestion
        result = self.run_cmd(cmd, required_real_result=True).strip()
```

And add  a test case? 

https://github.com/python/cherry-picker/blob/68153a8f93746fc7d8e5f4cb82707ea5550530e0/cherry_picker/test_cherry_picker.py#L343